tests/test_transforms/test_embeddings/test_models/utils.py

```diff
@@ -0,0 +1,9 @@
+import numpy as np
+import pandas as pd
+
+
+def cut_nan_timestamps(df: pd.DataFrame) -> np.ndarray:
+    """Cut NaN timestamps. Is used when encoding full series."""
+    last_timestamp = max(np.where(~df.isna().all(axis=1))[0])
+    df = df[: last_timestamp + 1]
```

Comment:
I don't understand how it works here. Is `last_timestamp` integer? What index does `df` uses`? 

If we select by 0-based index let's clarify it by using `.iloc`. If we select by `df` index let's clarify it by using `.loc`.